Automattic/sensei
# Add filter for the student bulk action modal

### Changes proposed in this Pull Request

* Add filters to add more flexibility and enable adding custom bulk actions.

### Testing instructions

* Checkout `add/student-bulk-actions-filters`.
* `npm install`, `npm run start`.
* Go to Sensei LMS -> Students, try different bulk actions.
* Bulk actions must work as expected.
* 
### New/Updated Hooks

* `senseiStudentActionMenuControls` - Filters controls for the single student action menu.
* `senseiStudentBulkActionModal` - Filters the bulk action modal.

### Screenshot / Video
